### PR TITLE
Expand the voucher report

### PR DESCRIPTION
**Pull request by marjev**
@mjfrey please take a look.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @marjev! It looks like you're a member of a company that does contract work for edX. If you're doing this work as part of a paid contract with edX, you should talk to edX about who will review this pull request. If this work is not part of a paid contract with edX, then you should ensure that there is an OSPR issue to track this work in [JIRA](https://openedx.atlassian.net), so that we don't lose track of your pull request.

To automatically create an OSPR issue for this pull request, just visit this link: https://openedx-webhooks.herokuapp.com/github/process_pr?repo=edx%2Fecommerce&number=538

**Comment by mjfrey** [comment]
@vkaracic Please take a look

File: ecommerce/extensions/voucher/tests/test_utils.py [hunk 1]
```diff
@@ -31,17 +35,19 @@ class UtilTests(TestCase):
...
-            product_class=self.coupon_product_class,
-            title='Test product'
-        )
+        self.course = CourseFactory()
+        self.verified_seat = self.course.create_or_update_seat('verified', False, 0, self.partner)
+
+        self.catalog = Catalog.objects.create(partner=self.partner)
 
-        self.stock_record = factories.create_stockrecord(self.coupon, num_in_stock=2)
+        self.stock_record = factories.create_stockrecord(self.verified_seat, num_in_stock=1)
```
Comment (vkaracic) [inline ecommerce/extensions/voucher/tests/test_utils.py]: I believe `create_or_update_seat` already creates a stock record for the seat: https://github.com/edx/ecommerce/blob/master/ecommerce/courses/models.py#L194-L215

File: ecommerce/extensions/voucher/tests/test_views.py [hunk 2]
```diff
@@ -9,17 +12,30 @@
...
     def setUp(self):
         super(CouponReportCSVViewTest, self).setUp()
+
+        self.user = self.create_user(full_name="Test User", is_staff=True)
+        self.client.login(username=self.user.username, password=self.password)
+
+        self.course = CourseFactory()
+        self.verified_seat = self.course.create_or_update_seat('verified', False, 0, self.partner)
+
+        self.stock_record = factories.create_stockrecord(self.verified_seat, num_in_stock=1)
```
Comment (vkaracic) [inline ecommerce/extensions/voucher/tests/test_views.py]: Same thing as above.

**Comment by vkaracic** [comment]
:+1:

**Comment by marjev** [comment]
@mjfrey I addressed @vkaracic's comments. Once the tests pass, you can take another look at this, just to make sure it's ready to be merged. Thanks!

**Comment by mjfrey** [comment]
:+1:

**Pull request merged**